### PR TITLE
Admin dashboard

### PR DESCRIPTION
**Pull request by kristi-h**

### CODE REVIEW

File: client/app/admin-dashboard/page.jsx [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
-                <ReservationTable />
-            </Suspense>
-        </div>
-    );
+  const appointments = await fetchAppointments();
+  return (
+    <div>
+      <Typography variant="h1">Reservations:</Typography>
+      <Suspense fallback={<CircularProgress />}>
+        <ReservationTable />
```
Comment (DalySoftware) [inline client/app/admin-dashboard/page.jsx:13]: ReservationTable is still making a separate fetch internally. I think it'd make more sense if it also accepted an appointments prop

File: client/components/admin_dashboard/.Map.jsx.swp
Comment (DalySoftware) [inline client/components/admin_dashboard/.Map.jsx.swp:1]: I don't think this is supposed to be committed

File: server/src/controllers/appointments.controller.js [hunk 2]
```diff
@@ -1,6 +1,27 @@
...
+    const data = await resp.json();
+    console.log(data);
+    if (data.status === "OK") {
+      const geodata = data.results[0].geometry.location;
+      console.log("geodata", geodata);
+
+      return geodata;
+    } else {
+      throw new Error(`Geocoding issue: ${data.status}`);
+    }
```
Comment (DalySoftware) [inline server/src/controllers/appointments.controller.js:18]: Quite weird to throw an error then immediately re-catch it. Would suggest if you want to throw something like this, it should be outside the try catch

**Review by DalySoftware — CHANGES_REQUESTED** [review]

**Review by DalySoftware — DISMISSED** [review]

**Review by DalySoftware — APPROVED** [review]

**Comment by tdkent** [comment]
I can assume it's all working,  but since the Google API key(s) have not been shared, I can't see the map or test the geocoding

**Pull request merged**